device: '/dev/ttyUSB0'
baudrate: 115200
debug: false
rate: 10 # Hz

slaves:
    module:
        id: 1
        leds:
            camera_led:
                id: 0
                timChId: 2
                smart: true
        bcs:
            manipulator:
                id: 0
                timChId: 0
                dirGPIOChId: 1
                disGPIOChId: 0
                termGPIOChId: 2
                speed: 100
        ms5837: true
